### DISCUSSION TITLE
240315

### DISCUSSION THREAD

**Discussion by jisunipark** [codeit-part4-8team-project/main#46] [팀회의]
## 멘토님 질문 사항

**Comment by sobokki** [comment]
## 논의사항

## 어제 한 일
mainpage 캘린더 부분 UI작업 
## 오늘 할 일
멘토링때 받은 솔루션 방향으로 캘린더 코드 리팩토링 하기 
## 오늘의 일정 (코어 타임)
## 멘토님 질문 사항

**Comment by jisunipark** [comment]
## 논의사항
- 아이콘 svg 파일 컴포넌트화
- 기본 레이아웃 공통 컴포넌트 분리
   - https://github.com/codeit-part4-8team-project/main/pull/44

## 어제 한 일
- 카드 컴포넌트 없을 시 보여줄 ui 작업
- 기본 레이아웃 공통 컴포넌트 분리

## 오늘 할 일
- 그룹 페이지 레이아웃
- 칸반보드 페이지 레이아웃

## 오늘의 일정 (코어 타임)
- x

## 멘토님 질문 사항

**Reply by jisunipark** [reply]
아이콘 svg 파일 컴포넌트화 -> 진행

**Comment by caseV12** [comment]
## 논의사항

## 어제 한 일
어제 일이 있어 작업을 하나도 못했습니다...
## 오늘 할 일
조금씩 리팩터링하는 거 말고 cors 해결 전에 할 게 없습니다. 
## 오늘의 일정 (코어 타임)
없습니다
## 멘토님 질문 사항

**Reply by jisunipark** [reply]
svg 파일 컴포넌트화

**Reply by MoonPillGyeom** [reply]
축하합니다 오늘 하나 생기셨습니다

**Reply by caseV12** [reply]
ㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋㅋ

**Comment by MoonPillGyeom** [comment]
## 논의사항
xx
## 어제 한 일
hookform 연동
## 오늘 할 일
컴포넌트 정리 및 hookform제어컴포넌트? 만들기
## 오늘의 일정 (코어타임)
xx
## 멘토님 질문 사항
xx
